.github/files/lambda-function/Makefile

```diff
@@ -0,0 +1,2 @@
+build-MartinFunction:
```

Comment:
Implicitly by `sam build` of the AWS::Serverless::Function. It _requires_ a Makefile. (Or like, something such as Java or JavaScript or Python which it knows how to build.)

(But AWS::Serverless::LayerVersion is happy to just zip up a directory. Which is a little bit silly since that's the complicated one and the function code is just one yaml, but oh well.)

See: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/building-custom-runtimes.html and https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/building-rust.html

Note that they also have a `cargo lambda` utility which is in beta, but might be worth a look later.